src/main/java/me/zeroeightsix/kami/module/modules/render/BreakingESP.kt

```diff
@@ -70,19 +81,37 @@ class BreakingESP : Module() {
         }
     }
 
+    private var warn = false
+    private var delay = 0
+
+    override fun onRender() {
+        if (screenWarn.value && warn) {
+            if (delay++ > 100) warn = false
+            val textRenderer = Wrapper.fontRenderer
+            val divider = DisplayGuiScreen.getScale()
```

Comment:
auto gui scale returns 0 so you should have a safety for that iirc

Reply:
Oh, good to know, ill change that.